frontend/src/concepts/pipelines/content/configurePipelinesServer/PipelineDropdown.tsx

```diff
@@ -94,14 +95,14 @@ export const PipelineDropdown = ({
           <MenuGroup
             label={
               <h1 className={css(styles.menuGroupTitle)}>
-                <KeyIcon /> Populate the form with credentials from your selected data connection
+                <KeyIcon /> Populate the form with credentials from your selected connection type
```

Comment:
```suggestion
                <KeyIcon /> Populate the form with credentials from your selected connection
```

I assume this is created connection, not a connection type, since that would almost always just be s3 on this page?